modules/python/test/test_copytomask.py

```diff
@@ -7,8 +7,8 @@
 # Python 2/3 compatibility
 from __future__ import print_function
 
-import cv2 as cv
 import numpy as np
+import cv2 as cv
```

Comment:
BTW, Could you share details about configuration to reproduce problem with ordering?

Reply:
there is a failed build on Custom node: https://pullrequest.opencv.org/buildbot/builders/precommit_custom_linux/builds/4489. But seems like it's not about order.

Reply:
Looks like this problem is related to different numpy versions (CentOS may have old version of numpy)